gadgets/Cargo.toml

```diff
@@ -54,7 +54,7 @@ version = "1.0.53"
 version = "2"
 
 [dependencies.digest]
-version = "0.9"
+version = "0.10"
 
```

Comment:
Can you check if `digest` can be a `dev-dep`?

Reply:
Done 👍